circle.yml

```diff
@@ -9,6 +9,7 @@ dependencies:
     - "~/examples/fsl_course_data"
   override:
     - pip install --upgrade pip
+    - pip install -r requirements.txt
```

Comment:
Why do we need this? All dependencies should be in the setup.py file and picked up by

	pip install -e .
	
I recommend removing requirements.txt from circle.yml. Keeping it may lead to tests passing, but pip-installed packages failing to run because some dependency was in in requirements.txt, but was left out of setup.py.